### ISSUE TITLE
[1.9] /tpworld is Broken for Sponge Bleeding

### ISSUE THREAD

**Issue by unitylinux** [hsyyid/EssentialCmds#95]
/tpworld is Broken for Sponge Bleeding

![tpworld-dim1](https://cloud.githubusercontent.com/assets/16209334/15035485/3b7807f0-1237-11e6-9030-2eb743cfd1c4.PNG)

Just to let you know.. in case you wanted to start a 1.9 Branch. /home and /sethome works along with a lot of other commands as well. I haven't had time to test all.

**Comment by jmiahman** [comment]
Here's the output from the SpongeVanilla(Bleeding) Server for this issue..

```
> sponge version                                                                
[15:06:27 INFO] [essentialcmds]: [Server] executed command sponge version       
[15:06:27 INFO]: SpongeVanilla                                                  
    Minecraft: 1.9                                                              
    SpongeAPI: 5.0.0-SNAPSHOT                                                   
    SpongeVanilla: 1.9-5.0.0-BETA-0          
```

```
[14:48:19 ERROR] [Sponge]: Error occurred while executing command 'tpworld DIM-1                                                         ' for source EntityPlayerMP['JMiahMan'/226, l='world', x=311.53, y=52.00, z=223.                                                         62]: org.spongepowered.api.entity.living.player.Player.transferToWorld(Ljava/lan                                                         g/String;Lcom/flowpowered/math/vector/Vector3d;)Z
java.lang.NoSuchMethodError: org.spongepowered.api.entity.living.player.Player.t                                                         ransferToWorld(Ljava/lang/String;Lcom/flowpowered/math/vector/Vector3d;)Z
        at io.github.hsyyid.essentialcmds.cmdexecutors.world.WorldsBase$Teleport                                                         .execute(WorldsBase.java:564) ~[WorldsBase$Teleport.class:8.1.0]
        at io.github.hsyyid.essentialcmds.cmdexecutors.world.TeleportWorldExecut                                                         or.execute(TeleportWorldExecutor.java:29) ~[TeleportWorldExecutor.class:8.1.0]
        at org.spongepowered.api.command.spec.CommandSpec.process(CommandSpec.ja                                                         va:331) ~[CommandSpec.class:1.9-5.0.0-BETA-0]
        at org.spongepowered.api.command.dispatcher.SimpleDispatcher.process(Sim                                                         pleDispatcher.java:331) ~[SimpleDispatcher.class:1.9-5.0.0-BETA-0]
        at org.spongepowered.common.command.SpongeCommandManager.process(SpongeC                                                         ommandManager.java:268) [SpongeCommandManager.class:1.9-5.0.0-BETA-0]
        at net.minecraft.command.ServerCommandManager.func_71556_a(SourceFile:80                                                         ) [bc.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_147361_d(SourceFile:8                                                         30) [mb.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_147354_a(SourceFile:8                                                         17) [mb.class:?]
        at net.minecraft.network.play.client.CPacketChatMessage.func_148833_a(So                                                         urceFile:37) [ij.class:?]
        at net.minecraft.network.play.client.CPacketChatMessage.func_148833_a(So                                                         urceFile:9) [ij.class:?]
        at org.spongepowered.common.network.PacketUtil.onProcessPacket(PacketUti                                                         l.java:117) [PacketUtil.class:1.9-5.0.0-BETA-0]
        at net.minecraft.network.PacketThreadUtil$1.redirect$onProcessPacket$0(S                                                         ourceFile:39) [fh$1.class:?]
        at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) [fh$1.cla                                                         ss:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:51                                                         1) [?:1.8.0_91]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_91]
        at net.minecraft.util.Util.func_181617_a(SourceFile:45) [g.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(SourceFile:132) [Mi                                                         necraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(SourceFil                                                         e:339) [la.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(SourceFile:553) [Mi                                                         necraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(SourceFile:457) [MinecraftSe                                                         rver.class:?]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_91]
```

**Comment by hsyyid** [comment]
Closed in 1114663e1f223f70cdac9c4a6268a143b6f55f03
